src/traces/box/calc.js

```diff
@@ -136,6 +136,11 @@ module.exports = function calc(gd, trace) {
             }
         };
 
+        // don't show labels in candlestick hover labels
+        if(trace._fullInput && trace._fullInput.type === 'candlestick') {
+            delete cd[0].t.labels;
+        }
```

Comment:
not a big deal, but 🐎  I might invert this logic and only add labels if we *do* want them.

Reply:
I thought about that. But, I prefer writing candlestick-only blocks like this one as _exceptions_ to a rule (the rule here being fill up `cd[0].t.labels`). Here's another [example](https://github.com/plotly/plotly.js/blob/55c4b3521c0cd426de6a52582f69852233c0284e/src/plots/cartesian/type_defaults.js#L121). 

I can't wait to rewrite `candlestick` and `ohlc` as good old (non-transform) trace types :upside_down_face:

Reply:
>  I prefer writing candlestick-only blocks like this one as exceptions to a rule

Good point.